### ISSUE TITLE
Please, support multitrack one CD FLACs !!! =DD

### ISSUE THREAD

**Issue by apometron** [UniversalMediaServer/UniversalMediaServer#1452]
It is FLACs files with cue sheet builtin, UMS do not supports embedded cue sheets.

It would be possible? Who many time for general availability?

**Comment by apometron** [comment]
Foobar2k supports it.

**Comment by Nadahar** [comment]
The first you would have to do if anyone should look at implementing it is to provide a sample.

**Comment by apometron** [comment]
150M Glenn Miller - Music of Your Life- Best of Glenn Miller.flac
https://uploadbr.com/fce20016904a7448

Sorry for the late reply.

**Comment by apometron** [comment]
Is it what you are asking?

**Comment by Nadahar** [comment]
@apometron Yes, that's fine except that I don't know Portuguese so the download took some guessing. I wouldn't hold my breath that anyone will do this quickly though, but now it's registered as a wish if somebody feels like doing it.

UMS would have to transcode the file to make serving it possible. I guess FFmpeg can do it, but implementing it in UMS will still take some work.
